src/components/LikeCoinButtonWidgetV2/LikeCoinButtonWidgetV2.vue

```diff
@@ -192,6 +213,12 @@ export default {
         y: 130,
       };
     },
+    upgradeLabelX() {
+      return 198;
+    },
```

Comment:
I think the x should align with avatar
![image](https://user-images.githubusercontent.com/17264716/207803480-0f2de083-bede-4131-8909-075c650d651e.png)

Reply:
left align? seems too close to the like button/text though

Reply:
<img width="326" alt="image" src="https://user-images.githubusercontent.com/6198816/207810060-88ddde8a-b185-4a6b-a2c3-7e4fdf98f81c.png">
How about this

Reply:
How about `Upgrade`? Is it still align to the center?

Reply:
If not, probably need to add `textAlign: center` to `upgradeLabelStyle` and add a fixed width

Reply:
<img width="315" alt="image" src="https://user-images.githubusercontent.com/6198816/207821960-e93e907a-bbd7-4455-b795-b6d0b3fc4df2.png">
numbers should be align but visually seems 1 px off...

Reply:
Probably due to the uppercase U, but I ll accept that